### DISCUSSION TITLE
More features for text formatting

### DISCUSSION THREAD

**Discussion by ttbielski** [xournalpp/xournalpp#4512] [Ideas] [answered]
It would be great if you could add more control for text formatting like in WYSIWYG editor:

- option to bold specific words instead of the whole text
- ability to turn words into links
- ability to change colors of specific words
- ability to paste code snippet from VSCode and keep the syntax highlighting

**Comment by netlimpopo** [comment, marked as answer]
The refactoring of the Text Editor is an important task. It has been requested by many users. Unfortunately it requires changes to the whole file format. So yes it will be tackled, but probably not in the very near future due to the complexity of the task. There is a possibility to use a codsyntax with the listings environment in LaTex, have you tried that?
Hope the issues can be adressed soon, we'd all be happy. 😅😀
